packages/react-core/src/next/components/Wizard/WizardNav.tsx

```diff
@@ -6,9 +6,9 @@ export interface WizardNavProps {
   /** children should be WizardNavItem components */
   children?: any;
   /** Aria-label applied to the nav element */
-  'aria-label'?: string;
+  ariaLabel?: string;
```

Comment:
No reason outside of not wanting to access them in the internal wizard's navigation via bracket notation `nav['aria-label']` vs `nav.ariaLabel`. I can revert since you're right in that elsewhere they are written as hyphenated and probably better to be consistent.

Reply:
I agree with keeping them consistent, since 'aria-label' is also the name of the HTML attribute